tests/system_tests/test_registries/test_link_registry.py

```diff
@@ -15,6 +15,9 @@ class LinkArtifactExpectation:
     collection: str = ""
 
 
+@pytest.mark.skip(
+    reason="LinkArtifact operation bypasses this codepath to avoid throwaway runs"
+)
```

Comment:
Delete instead of unconditionally skipping; we can always recover this from the git history if we wanted to.